### PR TITLE
en, zh: Add description about spec.toolImage

### LINKED ISSUE #1119: Add description about spec.toolImage in backup guiding
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
https://docs.pingcap.com/tidb-in-kubernetes/stable/backup-restore-overview, There is no description about how to specify the tool version. It's a new change in v1.1.9, and the related documents haven't been upgraded.
Here's the v1.1.9 release note.
https://docs.pingcap.com/tidb-in-kubernetes/stable/release-1.1.9

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### LINKED ISSUE #1145: 指定 dumpling 的版本
## Question

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Before submitting your question, make sure you have:

- Searched existing Stack Overflow questions.
- Googled your question.
- Read the documentation:
  - [TiDB Operator (en)](https://github.com/pingcap/docs-tidb-operator/tree/master/en)
  - [TiDB Operator (zh)](https://github.com/pingcap/docs-tidb-operator/tree/master/zh)

Now, please describe your question here:

问题：
请问使用dumpling 备份到 s3，如何指定 dumpling 的版本呢？[使用 Dumpling 备份 TiDB 集群数据到兼容 S3 的存储](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/backup-to-s3)，文档中好像没有找到

### PR DESCRIPTION
**Pull request by handlerww**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Closes #1145, Closes #1119

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by ran-huang** [comment]
Could you please involve a tech reviewer?

File: zh/backup-restore-overview.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 ### 通用字段介绍
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.toolImage`: 用于指定 `Backup` 使用的工具镜像, 例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`. (在 v1.1.9 以后版本支持这项配置)
```
Comment (DanielZhangQD) [inline zh/backup-restore-overview.md:46]:
```suggestion
* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像，例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，使用 BR 备份时默认使用 `pingcap/br:${tikv_version}` 进行备份，使用 Dumpling 备份时默认使用 https://github.com/pingcap/tidb-operator/blob/${operator_version}/images/tidb-backup-manager/Dockerfile 文件中 `TOOLKIT_V40` 指定的 Dumpling 版本进行备份。v1.1.9 及后续版本支持这项配置。
```

Comment (DanielZhangQD) [reply zh/backup-restore-overview.md:46]: Please also update the description for `Restore` CR.

Comment (handlerww) [reply zh/backup-restore-overview.md:46]: ok

**Comment by ran-huang** [comment]
@handlerww Can you update this PR?

File: zh/backup-restore-overview.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 ### 通用字段介绍
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像，例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，使用 BR 备份时默认使用 `pingcap/br:${tikv_version}` 进行备份，使用 Dumpling 备份时默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/${operator_version}/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_V40` 指定的 Dumpling 版本进行备份。v1.1.9 及后续版本支持这项配置。
```
Comment (csuzhangxc) [inline zh/backup-restore-overview.md:46]: `[Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/${operator_version}/images/tidb-backup-manager/Dockerfile)` 感觉这样写不太好，用户看到的是 `Backup Manager Dockerfile`，直接点链接又不能访问，还得点了后去手动修改地址栏

**Comment by qiancai** [comment]
/retitle en, zh: Add description about spec.toolImage 
/remove-translation doing
/translation done

File: zh/backup-restore-overview.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/backup-restore-overview.md:46]: master  就直接连接到 master 对应的代码吧

File: en/backup-restore-overview.md [hunk 3]
```diff
@@ -45,6 +45,7 @@ This section introduces the fields in the `Backup` CR.
 ### General fields
 
 * `.spec.metadata.namespace`: The namespace where the `Backup` CR is located.
+* `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not, BR will use the version in `pingcap/br:${tikv_version}` by default。 When using Dumpling to backup, the version specified in `TOOLKIT_V40` stored in [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/${operator_version}/images/tidb-backup-manager/Dockerfile) will be used by default. v1.1.9 and the later versions support this configuration.
```
Comment (DanielZhangQD) [inline en/backup-restore-overview.md:48]: Please replace the `。`.

File: zh/backup-restore-overview.md (hunk 2 above)
Comment (handlerww) [reply zh/backup-restore-overview.md:46]: fixed

File: en/backup-restore-overview.md (hunk 3 above)
Comment (handlerww) [reply en/backup-restore-overview.md]: fixed

File: zh/backup-restore-overview.md [hunk 4]
```diff
@@ -43,6 +43,7 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 ### 通用字段介绍
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像，例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，使用 BR 备份时默认使用 `pingcap/br:${tikv_version}` 进行备份，使用 Dumpling 备份时默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_V40` 指定的 Dumpling 版本进行备份。v1.1.9 及后续版本支持这项配置。
```
Comment (DanielZhangQD) [inline zh/backup-restore-overview.md:46]:
```suggestion
* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像，例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，使用 BR 备份时默认使用 `pingcap/br:${tikv_version}` 进行备份，使用 Dumpling 备份时默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_VERSION` 指定的 Dumpling 版本进行备份。v1.1.9 及后续版本支持这项配置。
```

File: en/backup-restore-overview.md [hunk 5]
```diff
@@ -45,6 +45,7 @@ This section introduces the fields in the `Backup` CR.
 ### General fields
 
 * `.spec.metadata.namespace`: The namespace where the `Backup` CR is located.
+* `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not, BR will use the version in `pingcap/br:${tikv_version}` by default. When using Dumpling to backup, the version specified in `TOOLKIT_V40` stored in [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used by default. v1.1.9 and the later versions support this configuration.
```
Comment (DanielZhangQD) [inline en/backup-restore-overview.md:48]:
```suggestion
* `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not set, `pingcap/br:${tikv_version}` will be used for backup with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for backup with Dumpling by default. v1.1.9 and the later versions support this configuration.
```

File: en/backup-restore-overview.md [hunk 6]
```diff
@@ -224,6 +225,7 @@ To restore data to a TiDB cluster in Kubernetes, you can create a `Restore` CR o
 This section introduces the fields in the `Restore` CR.
 
 * `.spec.metadata.namespace`: The namespace where the `Restore` CR is located.
+* `.spec.toolImage`：set the tools image used by `Restore`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/lightning:v4.0.10` when using Lightning. If not set, `pingcap/br:${tikv_version}` will be used for backup with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for backup with Lightning by default. v1.1.9 and the later versions support this configuration.
```
Comment (DanielZhangQD) [inline en/backup-restore-overview.md:228]:
```suggestion
* `.spec.toolImage`：set the tools image used by `Restore`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/lightning:v4.0.10` when using Lightning. If not set, `pingcap/br:${tikv_version}` will be used for a restore with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for a restore with Lightning by default. v1.1.9 and the later versions support this configuration.
```

Comment (DanielZhangQD) [reply en/backup-restore-overview.md:228]: Please update all the versions to 5.0.1 in master.

Comment (handlerww) [reply en/backup-restore-overview.md:228]: ok

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/backup-restore-overview.md [hunk 7]
```diff
@@ -43,6 +43,7 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 ### 通用字段介绍
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像，例如使用 BR 备份时可以设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Dumpling 备份时设置为 `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，使用 BR 备份时默认使用 `pingcap/br:${tikv_version}` 进行备份，使用 Dumpling 备份时默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_VERSION` 指定的 Dumpling 版本进行备份。v1.1.9 及后续版本支持这项配置。
```
Comment (qiancai) [inline zh/backup-restore-overview.md:46]:
```suggestion
* `.spec.toolImage`：用于指定 `Backup` 使用的工具镜像。
    - 使用 BR 备份时，可以用该字段指定 BR 的版本，例如，`spec.toolImage: pingcap/br:v4.0.10`。如果不指定，默认使用 `pingcap/br:${tikv_version}` 进行备份。
    - 使用 Dumpling 备份时，可以用该字段指定 Dumpling 的版本，例如， `spec.toolImage: pingcap/dumpling:v4.0.10`。如果不指定，默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_VERSION` 指定的 Dumpling 版本进行备份。           
    - TiDB Operator 从 v1.1.9 版本起支持这项配置。
```

File: en/backup-restore-overview.md [hunk 8]
```diff
@@ -45,6 +45,7 @@ This section introduces the fields in the `Backup` CR.
 ### General fields
 
 * `.spec.metadata.namespace`: The namespace where the `Backup` CR is located.
+* `.spec.toolImage`：set the tools image used by `Backup`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/dumpling:v4.0.10` when using Dumpling. If not set, `pingcap/br:${tikv_version}` will be used for backup with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for backup with Dumpling by default. v1.1.9 and the later versions support this configuration.
```
Comment (qiancai) [inline en/backup-restore-overview.md:48]:
```suggestion
* `.spec.toolImage`: The tool image used by `Backup`.
    - When using BR for backup, you can specify the BR version in this field. For example, `.spec.toolImage: pingcap/br:v4.0.10`. If not specified, `pingcap/br:${tikv_version}` is used for backup by default.
    - When using Dumpling for backup, you can specify the Dumpling version in this field. For example, `spec.toolImage: pingcap/dumpling:v4.0.10`. If not specified, the Dumpling version specified in `TOOLKIT_VERSION` of the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) is used for backup by default.          
    - TiDB Operator supports this configuration starting from v1.1.9.
```

File: zh/backup-restore-overview.md [hunk 9]
```diff
@@ -211,6 +212,7 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 为了对 Kubernetes 上的 TiDB 集群进行数据恢复，用户可以通过创建一个自定义的 `Restore` Custom Resource (CR) 对象来描述一次恢复，具体恢复过程可参考 [备份与恢复简介](#备份与恢复简介)中列出的文档。以下介绍 Restore CR 各个字段的具体含义。
 
 * `.spec.metadata.namespace`：`Restore` CR 所在的 namespace。
+* `.spec.toolImage`：用于指定 `Restore` 使用的工具镜像，例如使用 BR 恢复时设置为 `spec.toolImage: pingcap/br:v4.0.10` 或者在使用 Lightning 恢复时可以设置为 `spec.toolImage: pingcap/lightning:v4.0.10`。如果不指定，使用 BR 恢复时默认使用 `pingcap/br:${tikv_version}` 进行恢复，使用 Lightning 恢复时默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_VERSION` 指定的 Lightning 版本进行恢复。v1.1.9 及后续版本支持这项配置。
```
Comment (qiancai) [inline zh/backup-restore-overview.md:215]:
```suggestion
* `.spec.toolImage`：用于指定 `Restore` 使用的工具镜像。
    - 使用 BR 恢复时，可以用该字段指定 BR 的版本。例如，`spec.toolImage: pingcap/br:v4.0.10`。如果不指定，默认使用 `pingcap/br:${tikv_version}` 进行恢复。
    - 使用 Lightning 恢复时，可以用该字段指定 Lightning 的版本，例如`spec.toolImage: pingcap/lightning:v4.0.10`。如果不指定，默认使用 [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) 文件中 `TOOLKIT_VERSION` 指定的 Lightning 版本进行恢复。
    - TiDB Operator 从 v1.1.9 版本起支持这项配置。
```

File: en/backup-restore-overview.md [hunk 10]
```diff
@@ -224,6 +225,7 @@ To restore data to a TiDB cluster in Kubernetes, you can create a `Restore` CR o
 This section introduces the fields in the `Restore` CR.
 
 * `.spec.metadata.namespace`: The namespace where the `Restore` CR is located.
+* `.spec.toolImage`：set the tools image used by `Restore`. For example, you can set `spec.toolImage: pingcap/br:v4.0.10` when using BR, or you can set `spec.toolImage: pingcap/lightning:v4.0.10` when using Lightning. If not set, `pingcap/br:${tikv_version}` will be used for a restore with BR by default, and the `TOOLKIT_VERSION` in the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) will be used for a restore with Lightning by default. v1.1.9 and the later versions support this configuration.
```
Comment (qiancai) [inline en/backup-restore-overview.md:228]:
```suggestion
* `.spec.toolImage`：The tools image used by `Restore`. 
    - When using BR for restoring, you can specify the BR version in this field. For example,`spec.toolImage: pingcap/br:v4.0.10`. If not specified, `pingcap/br:${tikv_version}` is used for restoring by default.
    - When using Lightning for restoring, you can specify the Lightning version in this field. For example, `spec.toolImage: pingcap/lightning:v4.0.10`. If not specified, the Lightning version specified in `TOOLKIT_VERSION` of the [Backup Manager Dockerfile](https://github.com/pingcap/tidb-operator/blob/master/images/tidb-backup-manager/Dockerfile) is used for restoring by default. 
    - TiDB Operator supports this configuration starting from v1.1.9. 
```

**Comment by qiancai** [comment]
@handlerww, PTAL. Thanks.

File: zh/backup-restore-overview.md [hunk 11]
```diff
@@ -211,6 +215,10 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 为了对 Kubernetes 上的 TiDB 集群进行数据恢复，用户可以通过创建一个自定义的 `Restore` Custom Resource (CR) 对象来描述一次恢复，具体恢复过程可参考 [备份与恢复简介](#备份与恢复简介)中列出的文档。以下介绍 Restore CR 各个字段的具体含义。
 
 * `.spec.metadata.namespace`：`Restore` CR 所在的 namespace。
+* `.spec.toolImage`：用于指定 `Restore` 使用的工具镜像。
+    - 使用 BR 恢复时，可以用该字段指定 BR 的版本。例如，`spec.toolImage: pingcap/br:v4.0.10`。如果不指定，默认使用 `pingcap/br:${tikv_version}` 进行恢复。
```
Comment (DanielZhangQD) [inline zh/backup-restore-overview.md:219]: 版本要改成 5.0.1

Comment (handlerww) [reply zh/backup-restore-overview.md]: 改了

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**